consultation_analyser/consultations/jinja2/consultations/answers/index.html

```diff
@@ -38,6 +38,60 @@ <h2 class="govuk-heading-m govuk-!-font-weight-regular">
   </div>
 
   <div class="govuk-grid-row govuk-!-margin-top-5">
+    <div class="iai-overview-container">
```

Comment:
Should we also add a class of `govuk-grid-column-full` to make the padding/width the same as the rest of the page?

<img width="720" alt="Screenshot 2025-04-08 at 14 24 09" src="https://github.com/user-attachments/assets/ac9fe430-d05d-44df-924d-c325b5e339a1" />

Happy to defer to you, Kevin and Charlotte if this is intentionally wider.